README.md

```diff
@@ -18,63 +18,34 @@
 # under the License.
 #
 -->
-## Mobile Spec Suite ##
+# Cordova Mobile Spec Suite
 
-This project is a set of automated & manual tests that test
-Cordova core functionality.
+This `cordova-mobile-spec` repository can be used to create a Cordova app that provides a set of manual tests to verify Cordova core functionality.  
```

Comment:
```suggestion
This `cordova-mobile-spec` repository can be used to create a Cordova app that provides a set of manual tests to verify Cordova core functionality, especially on plugin releases and platform releases.
```

Clarification of information that was not clear to me until yesterday or maybe the day before. May need rewording.

Reply:
This is not correct. 

One could possibly write "It is used in the 'verification'  steps of plugin and platform releases" if it is really mentioned in both release process documents.

Reply:
From `grep` I find `mobilespec` in the following cordova-coho docs:
* `docs/platforms-release-process.md`
* `docs/plugins-release-process.md`
* `tools-release-process.md`

So clarification that we are supposed to use the same test tool in all of these release processes would be nice.

Reply:
Yep, added it as it reflects the current documentation.
Will have to see if it stays that way tbh - mobilespec might not be the right tool for it. Will have to evaluate when I understood the rest of our test tooling (Appium etc) - will create new issues/PRs then.

Reply:
Makes sense, +1 for using the word "currently"